src/borg/archiver.py

```diff
@@ -1024,7 +1028,11 @@ def do_prune(self, args, repository, manifest, key):
                     else:
                         if args.output_list:
                             list_logger.info('Pruning archive: %s' % format_archive(archive))
-                        Archive(repository, key, manifest, archive.name, cache).delete(stats, forced=args.forced)
+                        Archive(repository, key, manifest, archive.name, cache,
+                                progress=args.progress).delete(stats, forced=args.forced)
+                        if args.progress:
+                            archives_deleted += 1
+                            logger.warning('(%d/%d) archives pruned' % (archives_deleted, to_delete_len))
```

Comment:
this is progress display (information), not a warning.

why didn't you use the ProgressIndicator used elsewhere?